docs/developers/debugging/releases.md

```diff
@@ -10,6 +10,26 @@
   locally, just symlink your source dir as `c` in an empty
   directory (versus checking out).
 
+## Mapping releases back to git commits
+
+The source IREE commit SHA is embeded into pip releases in a few places.
+Starting in a python venv, you can find the IREE commit from both the shell:
+
+```shell
+"$(find . -name 'iree-compile' -executable)" --version
+IREE (https://openxla.github.io/iree):
+  IREE compiler version 20231016.553 @ f1cb2692a086738d7f16274b9b3af6d2c15ef133
+  LLVM version 18.0.0git
+  Optimized build
+```
+
+and the Python API:
+
+```shell
+python -c "import iree.compiler.version as v; print(v.REVISIONS['IREE'])"
+f1cb2692a086738d7f16274b9b3af6d2c15ef133
+``` 
+
```

Comment:
Maybe file an issue for iree.runtime.version being broken and note what should work here?

Reply:
Filed here: https://github.com/openxla/iree/issues/15213